notebooks/103-paddle-to-openvino/103-paddle-to-openvino-classification.ipynb

```diff
@@ -11,9 +11,9 @@
     "\n",
```

Comment:
<p>See the&nbsp;<a href="https://docs.openvino.ai/2023.0/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html" target="_blank">Model Optimizer Developer Guide</a>&nbsp;for more information about Model Optimizer. - please, change to model conversion API https://docs.openvino.ai/2023.0/openvino_docs_model_processing_introduction.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Done</p>

Reply:
<p>no, "about Model Optimizer." at the end is still exist</p>